### PR TITLE
Improve NxToggle focus style RSC-602

### PR DESCRIPTION
**Pull request by avnvbhatta**
https://issues.sonatype.org/browse/RSC-602

- Added focus styles on `NxToggle` that shows a light outer blue border when focused.
- Modified visual tests to align with new design spec.

Achieved by using an absolutely positioned `::after` pseudoelement that is visible on focus.

### CODE REVIEW

File: gallery/visualtests/nxToggle.js [hunk 1]
```diff
@@ -56,7 +61,8 @@ describe('NxToggle', function() {
       }
     });
 
-    it('dark border, blue background and white indicator when clicked, focused, and hovered', async function() {
+    it(`dark border, blue background and white indicator with light outer blue border
```
Comment (rpokorny) [inline gallery/visualtests/nxToggle.js:64]: The hover border is black on these actually

File: lib/src/components/NxToggle/NxToggle.scss [hunk 2]
```diff
@@ -31,12 +32,22 @@
...
+        border: 1px solid var(--nx-color-interactive-border-focus);
+        border-radius: 16px;
+        box-shadow: var(--nx-box-shadow-focus);
+        content: '';
+        display: block;
+        height: 30px;
+        position:absolute;
+        left: -4px;
+        top: -4px;
+        width: 50px;
```
Comment (rpokorny) [inline lib/src/components/NxToggle/NxToggle.scss:49]: Should we turn the `height` and `width` of the control into local scss variables and derive the focus border sizes from it?

Comment (avnvbhatta) [reply lib/src/components/NxToggle/NxToggle.scss:49]: I think it's a good idea, but I think would it make sense to do it under a different JIRA ticket and use variables for focus borders of radios and checkboxes as well?

Comment (rpokorny) [reply lib/src/components/NxToggle/NxToggle.scss:49]: > I think it's a good idea, but I think would it make sense to do it under a different JIRA ticket and use variables for focus borders of radios and checkboxes as well?

Sure.  Put in a tech debt ticket and link it here.

Comment (avnvbhatta) [reply lib/src/components/NxToggle/NxToggle.scss:49]: > > I think it's a good idea, but I think would it make sense to do it under a different JIRA ticket and use variables for focus borders of radios and checkboxes as well?
> 
> Sure. Put in a tech debt ticket and link it here.

Tech Debt ticket: https://issues.sonatype.org/browse/RSC-620

**Review by rpokorny — APPROVED** [review]

**Pull request merged**